openapi/components/schemas/RiskScore/RiskScoreRules.yaml

```
@@ -0,0 +1,84 @@
+type: object
+description: Risk score rules.
+required:
+  - isProxy
+  - isVpn
+  - isTor
+  - isHosting
+  - hasMismatchedBillingAddressCountry
+  - hasMismatchedBankCountry
+  - hasMismatchedTimeZone
+  - hasMismatchedHolderName
+  - hasFakeName
+  - isHighRiskCountry
+  - paymentInstrumentVelocity
+  - declinedPaymentInstrumentVelocity
+  - deviceVelocity
+  - ipVelocity
+  - emailVelocity
+  - billingAddressVelocity
+properties:
+  isProxy:
+    description: Specifies whether the customer's IP address is related to a proxy.
+    allOf:
+      - $ref: "./RiskScoreBoolean.yaml"
```

```suggestion
      - $ref: ./RiskScoreBoolean.yaml
```

I think double quotes are excessive here